src/Install/Controller/InstallController.php

```diff
@@ -77,7 +77,7 @@ public function handle(Request $request): ResponseInterface
             'email'                 => array_get($input, 'adminEmail'),
         ]);
 
-        $baseUrl = rtrim((string) $request->getAttribute('originalUri'), '/');
+        $baseUrl = rtrim((string) $request->getAttribute('originalUri', $request->getUri()), '/');
```

Comment:
Can we simply use `$request->getUri()`? Not sure why we are getting the "original" URI, whatever that is.

Reply:
I guess we can, I was not sure what is `originalUri` either.

Reply:
@sijad would you be able to please try it out?

Reply:
I've tested `$baseUrl = rtrim((string) $request->getUri(), '/');` with clean install, everything seems fine, should I change this PR to remove `originalUri`?

Reply:
Yep. In fact, `originalUri` is the real problem here. It [was available in zend-stratigility v1](https://docs.zendframework.com/zend-stratigility/v2/migration/#original-request-response-and-uri), but apparently we forgot to adopt the v2 way when upgrading. Simply `$request->getUri()` will do. :+1:

Reply:
Correction: The `rtrim` thing should stay in place, so your last suggestion is just right.